sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/data/ExponentialHistogramPointData.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.data;
+
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * ExponentialHistogramPointData represents an approximate distribution of measurements across
+ * exponentially increasing bucket boundaries, taken for a {@link ExponentialHistogramData}. It
+ * contains the necessary information to calculate bucket boundaries and perform aggregation.
+ *
+ * <p>The bucket boundaries are calculated using both the scale {@link #getScale()}, and the offset
+ * {@link ExponentialHistogramBuckets#getOffset()}.
+ *
+ * <p>See:
+ * https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/datamodel.md#exponentialhistogram
+ */
+@Immutable
+public interface ExponentialHistogramPointData extends PointData {
+
+  /**
+   * Scale characterises the resolution of the histogram, with larger values of scale offering
+   * greater precision. Bucket boundaries of the histogram are located at integer powers of the
+   * base, where <code>base = Math.pow(2, Math.pow(2, -scale))</code>.
+   *
+   * @return the scale.
+   */
+  int getScale();
+
+  /**
+   * Returns the sum of all measurements in the data point. The sum should be disregarded if there
+   * are both positive and negative measurements.
+   *
+   * @return the sum of all measurements in this data point.
+   */
+  double getSum();
+
+  /**
+   * Returns the number of measurements taken for this data point, including the positive bucket
+   * counts, negative bucket counts, and the zero count.
+   *
+   * @return the number of measurements in this data point.
+   */
+  long getCount();
+
+  /**
+   * Returns the base, which is calculated via the scale {@link #getScale()}. The larger the base,
+   * the further away bucket boundaries are from each other.
+   *
+   * <p><code>base = 2^(2^-scale)</code>
+   *
+   * @return the base.
+   */
+  default double getBase() {
```

Comment:
I don't see `base` in the OTLP proto, I suspect we don't need this as a public method on PointData (presumably we'll have a private method somewhere to compute this where needed)

Reply:
It's just for convenience / visibility. It doesn't necessarily need to be there. I can remove it if you feel strongly.

Reply:
I'm not sure that `base` is practically useful. In the logarithm implementation, there is a multiplicative factor `log2(E) * 2**scale` we find especially useful in order to use the builtin natural logarithm function.

Reply:
Alright, seems that folks would prefer to go without it, so I've removed it